back/app/controllers/omniauth_callback_controller.rb

```diff
@@ -94,26 +89,23 @@ def auth_callback(verify:, authver_method:)
           @user.save!
           @invite.save!
           SideFxInviteService.new.after_accept @invite
-          signup_success_redirect
+          verify_and_sign_in(auth, @user, verify, sign_up: true)
```

Comment:
I guess the main change to the "user created even though verification failed"-issue is here, right?

But it seems that the user is still created above, and that the only thing is that we don't set the token. Am I understanding that correctly?